mlflow/spark.py

```diff
@@ -725,6 +725,7 @@ def _load_model(model_uri, dfs_tmpdir_base=None, local_model_path=None):
     from pyspark.ml.pipeline import PipelineModel
 
     dfs_tmpdir = _tmp_path(dfs_tmpdir_base or MLFLOW_DFS_TMP.get())
+    _logger.warn(f"is_in_cluster? {databricks_utils.is_in_cluster()}, is_dbfs_fuse_available? {databricks_utils.is_dbfs_fuse_available()}")
```

Comment:
Result:
```
2022/12/08 21:02:01 WARNING mlflow.spark: is_in_cluster? False, is_dbfs_fuse_available? True
```

Reply:
@dbczumar @harupy Do you have any thoughts?

Reply:
<img width="1083" alt="image" src="https://user-images.githubusercontent.com/17039389/206628010-7ded047f-4acc-43da-83c3-56b46c518fc0.png">

Reply:
@harupy Thanks for taking a look. It is True when executing in the notebook, but the line I logged is from a subprocess that does not have a spark session.